contrib/check-config.sh

```diff
@@ -60,17 +60,20 @@ color() {
 			white) code=37 ;;
 		esac
 		if [ "$code" ]; then
-			codes=("${codes[@]}" "$code")
+			[ "$codes" ] && codes="$codes "
+			codes="$codes$code"
 		fi
 	fi
-	local IFS=';'
-	echo -en '\033['"${codes[*]}"'m'
+	oldIFS=$IFS
+	IFS=';'
+	printf '\033[%sm' "$codes"
+	IFS=$oldIFS
```

Comment:
Adjusting the instance above that adds a second code to use `;` instead of ` ` makes this not need `IFS` at all:

```suggestion
	printf '\033[%sm' "$codes"
```